scripts/asr_language_modeling/ngram_lm/eval_beamsearch_ngram.py

```diff
@@ -151,7 +152,11 @@ def beam_search_eval(
     model.cfg.decoding.beam = cfg.decoding
 
     # Update model's decoding strategy
-    model.change_decoding_strategy(model.cfg.decoding)
+    if isinstance(model, EncDecHybridRNNTCTCModel):
+        model.change_decoding_strategy(model.cfg.decoding, decoder_type='ctc')
+        model.decoding = model.ctc_decoding
+    else:
+        model.change_decoding_strategy(model.cfg.decoding)
```

Comment:
You may save the decoding or ctc_decoding based on the type of the model in a variable named decoding and just use that in the rest of the model instead of using model.decoding.

Reply:
moved it into inside change_decoding_strategy()

Reply:
It would overwrite the RNNT decoding. How about doing something like the following?
```
if isinstance(model, EncDecHybridRNNTCTCModel):
        model.change_decoding_strategy(model.cfg.decoding, decoder_type='ctc')
	decoding = model.ctc_decoding
    else:
        model.change_decoding_strategy(model.cfg.decoding)
	decoding = model.decoding

```

Reply:
done